Package@swift-5.3.swift

```diff
@@ -35,11 +35,14 @@ let package = Package(
             name: "AppCenter",
             path: "AppCenter/AppCenter",
             exclude: ["Support"],
-            cSettings: [
-                .define("APP_CENTER_C_VERSION", to:"\"4.4.2\""),
-                .define("APP_CENTER_C_BUILD", to:"\"1\""),
-                .headerSearchPath("**"),
-            ],
+            cSettings: {
```

Comment:
Hi! I checked your changes, but it doesn't work for me.
When I'm trying to call the `swift test` command (from the feature description)  I get the next error:
```
error: manifest parse error(s):
Unknown error parsing manifest for appcenter-sdk-apple 
```

Reply:
Oh, right, I need to update that. That was `swift test` in local projects that use AppCenter SDK. 

To test this for the project, you would need to specify `swift build`.

Reply:
My apologies, there was an error in the committed code. 

Could you please try running:

```sh
swift build --product AppCenterAnalytics
swift build --product AppCenterCrashes
```

These should work. I'm looking to see if I can make plain `swift build` work, but there's iOS code mixed in via the AppCenterDistribute target that isn't being excluded from the Swift package properly.

Reply:
Thanks for the update! 
Now it works for me with the `swift build` command, but fails with the `swift test` with error:
```
fatal error: 'UIKit/UIScreen.h' file not found
#import <UIKit/UIScreen.h>
        ^~~~~~~~~~~~~~~~~~
```

We will discuss these changes with the team and come back with the results.

Reply:
Thank you. I realise it's not an ideal solution to the problem, but AppCenter not working with SwiftPM outside of Xcode is an issue that's going to continue to grow as people migrate their projects to use more Swift packages.